testing/api/test_api.py

```diff
@@ -1444,3 +1695,52 @@ def test_delete_profile(testrun, reset_profiles, add_profile): # pylint: disable
   )
   # Check if profile was deleted
   assert deleted_profile is None
+
+def test_delete_profile_no_profile(testrun, reset_profiles): # pylint: disable=W0613
+  """Test delete profile if the profile does not exists"""
+  # Load the profile to delete
+  profile_to_delete = load_profile("new_profile.json")
+
+  # Delete the profile
+  r = requests.delete(
+      f"{API}/profiles",
+      data=json.dumps(profile_to_delete),
```

Comment:
The JSON sent to the delete request should just be the name field

Reply:
You could just generate your own JSON rather than loading in a file for something this small